Fixes SettingsPane's minimal size bug

LightTab2 opened the pull request:
Fixes #4.

`$SettingsPane/VBoxContainer` calculates its minimal size based only on its children.

LightTab2 committed:
Fixes #4

$SettingsPane/VBoxContainer calculates its minimal size based only on its children

albinaask commented:
A great many thanks for you taking the time to fix bugs. I'll get to pulling it as soon as I possibly can make time to.

albinaask committed:
Updated plugin.cfg and a related bug fix

- Realized that dialog could be shrunk to the point where the scroll container disappeared. This is no longer the case.
